### ISSUE TITLE
[Suggestion] Make it so it's possible to have account loaded in the config file

### ISSUE THREAD

**Issue by qfoxb** [GeyserMC/GeyserConnect#50]
It would obviously not work on a public server, but being able to set something in config to login with pre-saved credentials for a privately run geyserconnect instance would be convenient.

**Comment by rtm516** [comment]
This is supported in Geyser which intheory should have worked in the new GeyserConnect builds. But due to it causing some odd behaviour with connecting I disable the feature on startup.

Leaving this issue open incase the fix is worked out further down the line.

**Comment by severtheskyline** [comment]
> This is supported in Geyser which intheory should have worked in the new GeyserConnect builds. But due to it causing some odd behaviour with connecting I disable the feature on startup.
> 
> Leaving this issue open incase the fix is worked out further down the line.

Yeah, having so your account is flagged to be saved in the config file works when loading geyser standalone without geyserconnect, it stores it in a file and everything. But when loading geyserconnect it completely ignores the setting and you have to re-auth every time.
